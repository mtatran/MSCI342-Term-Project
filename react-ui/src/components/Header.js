import React, {Component, Fragment} from "react";
import logo from '../logo.svg'
import "./Header.scss"

export default class Header extends Component {
    render() {

      let header;
        header = (<header className="App-header-blue">
                    <img src={logo} className="App-logo" alt="logo" />
                 </header>)
      return (
        <Fragment>
       {header}
       </Fragment>
      );
    }
  }
