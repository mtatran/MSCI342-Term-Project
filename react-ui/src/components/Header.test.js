import React from "react";
import { create } from "react-test-renderer";
import { render, getByTestId} from '@testing-library/react';


import Header from './Header';

describe('Header', () => {
  test('snapshot renders', () => {
    const component = create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });

});