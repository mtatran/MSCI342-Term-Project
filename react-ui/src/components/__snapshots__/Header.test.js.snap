// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot renders 1`] = `
<header
  className="App-header-blue"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
</header>
`;
